### PR TITLE
LR-TD code base cleanup

### PR DESCRIPTION
**Pull request by vanderhe**
Trivial cosmetic changes only (excludes timeprop.F90).

### CODE REVIEW

File: src/dftbp/timedep/linrespgrad.F90 [hunk 1]
```diff
@@ -2315,16 +2313,16 @@ subroutine calcWvectorZ(zz, win, homo, getIA, getIJ, getAB, iaTrans, iAtomStart,
   end subroutine calcWvectorZ
 
 
-  !> Write out density matrix, full if rhoSqr is present
+  !> Write out density matrix, full if rhoSqr is present.
```
Comment (bhourahine) [inline src/dftbp/timedep/linrespgrad.F90:2316]:
```suggestion
  !> Write out density matrix, full matrix if rhoSqr is present, but transition part only if not.
```

**Review by aradi — APPROVED** [review]
LGTM

File: src/dftbp/timedep/linrespgrad.F90 [hunk 2]
```diff
@@ -3957,16 +3956,16 @@ subroutine grndToExcDensityMatrices(tHybridXc, xpy, xmy, win, iAtomStart, homo,
...
+    nxvv(:) = (nvir * (nvir + 1)) / 2
 
     allocate(vecHoo(sum(nxoo)))
 
-    !! transition charges use compound index ijs = ij + soo(s)
+    ! Transition charges use compound index ijs = ij + soo(s)
     soo(:) = [0, nxoo(1)]
 
-    p   = 0.0_dp
+    p = 0.0_dp
```
Comment (bhourahine) [inline src/dftbp/timedep/linrespgrad.F90:3968]: (:) for p, wov and woo here

**Review by bhourahine — COMMENTED** [review]
Some suggestions so far.

**Comment by vanderhe** [comment]
> Some suggestions so far.

Thanks, I've applied the suggestions.

**Pull request merged**